content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/_cq_design_dialog/.content.xml

```diff
@@ -31,9 +31,9 @@
                 sling:resourceType="granite/ui/components/coral/foundation/tabs"
                 maximized="{Boolean}true">
                 <items jcr:primaryType="nt:unstructured">
-                    <sizes
+                    <title
```

Comment:
@Shivanshmathur - the problem here is that the tabs are considered API, if we rename the node, we would have to bump the component version (the customer may have overriden that tab). See https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/wiki/Versioning-Policies#dialogs

That's why my original suggestion was to keep the node name the same and change the `jcr:title` - even though it's wrong semantically. What do you think?

Reply:
https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/issues/233